spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe Api::V1::UsersController do
+  before(:each) do
+    allow(Figaro.env).to receive(:transitland_datastore_auth_token) { 'THISISANOTHERKEY' }
+    @request.env['HTTP_AUTHORIZATION'] = 'Token token=THISISANOTHERKEY'
+  end
+
+  let(:user) { create(:user) }
+
+  context 'GET index' do
+    it 'requires auth key to view' do
+      @request.env['HTTP_AUTHORIZATION'] = nil
+      get :index
+      expect(response.status).to eq 401
+    end
+
+    it 'lists all users when auth key is provided' do
+      create_list(:user, 2)
+      get :index
+      expect_json_types({ users: :array })
+      expect_json({ users: -> (users) {
+        expect(users.length).to eq 2
+      }})
+    end
+  end
+
+  describe 'GET show' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.